food/parallel.js

```diff
@@ -692,18 +593,9 @@ function exclude_data() {
   rescale();
 }
 
-function remove_axis(d,g) {
```

Comment:
man... what is happening here?

Reply:
remove 1 dimension, rescale the domain for the shorter length of dimensions, shifting the existing ticks lines, removing the dimension from the the SVG (the ticks), and finally re-rendering D3.

Reply:
oh.... okay, I see now how that's happening. the tick lines are objects themselves that have to be placed on top of vertical lines. wow. of course. I get how this renders now... thank you for your patience in translating that.